### PR TITLE
Norwegian (Bokmål) localization.

### PR DESCRIPTION
**Pull request by SomeGreenGoo**
Adds norwegian (Bokmål) localization.

### CODE REVIEW

File: Assets/StreamingAssets/Localization/no-BM.lang [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+new_world=Ny verden
+load=Åpne
+save=Lagre
+temp_pathfinding_test=Pathfinding test
+selection_panel_title=Utvalg
+save_file=Lagre fil...
+load_file=Åpne fil...
+okay=OK
+cancel=Avbryt
+filename=Fil navn
```
Comment (henninglive) [inline Assets/StreamingAssets/Localization/no-BM.lang]: `Filnavn` should be written as single word.

**Comment by WardBenjamin** [comment]
Other than the note from @henninglive, looks good (not that i know much Norwegian :P)

File: Assets/StreamingAssets/Localization/no-BM.lang [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+furn_steel_wall=Stålvegg
+furn_steel_wall_desc=En vegg av stål.
+furn_door=Dør
+furn_door_desc=En dør.
+furn_stockpile=Lager
+furn_stockpile_desc=Lagringsplass for dine ressurser.
+furn_oxygen_generator=Oksygengenerator
+furn_oxygen_generator_desc=Produserer oksygen for astronautene dine.
+furn_mining_drone_station=Gruve
+furn_mining_drone_station_desc=Lager resurser.
```
Comment (NicTorgersen) [inline Assets/StreamingAssets/Localization/no-BM.lang]: `Ressurser` should be written with double `s`.

**Comment by henninglive** [comment]
Looks good.

**Pull request merged**